README.md

```diff
@@ -41,6 +41,18 @@ Next, we will need to install `wasm-bindgen-cli`. In the future Trunk will handl
 cargo install wasm-bindgen-cli
 ```
 
+In addition, we will need to install `wasm-opt` which is part of `binaryen`. On MacOS we can install
```

Comment:
Per my comments below, until we get to the point where we are actually downloading wasm-opt for the user, I would say that we should update this verbiage to something like: `If using wasm-opt, we will need to install ...`.

Reply:
Definitely. Have been a bit occopied this week but I'll continue on this soon and adjust to your comments.